universal-login-react/src/ui/styles/themes/Jarvis/walletSelectorThemeJarvis.sass

```diff
@@ -10,7 +10,7 @@
     font-size: 14px
     color: #31405E
 
-  .input-unilogin-component-input-wallet-selector
+  .unilogin-component-input-wallet-selector
```

Comment:
Where this className is created? Because in `Input.tsx` you are creating `.unilogin-component-input wallet-selector` and IMO there will be no such class name as `.unilogin-component-input-wallet-selector`

Reply:
Ok, sorry. I checked it in the code and this class name is created in `EnsNamePicker.tsx`